apps/meteor/app/lib/lib/MessageTypes.ts

```diff
@@ -310,6 +311,73 @@ Meteor.startup(function () {
 			};
 		},
 	});
+	MessageTypes.registerType({
+		id: 'video-direct-calling',
+		system: true,
+		message: 'video_direct_calling',
+		data(message) {
+			return {
+				username: message.u.username,
+			};
+		},
+	});
+	MessageTypes.registerType({
+		id: 'video-direct-missed',
+		system: true,
+		message: 'video_direct_missed',
+		data(message) {
+			return {
+				username: message.u.username,
+			};
+		},
+	});
+	MessageTypes.registerType({
+		id: 'video-direct-started',
+		system: true,
+		message: 'video_direct_started',
+		data(message) {
+			return {
+				username: message.u.username,
+			};
+		},
+	});
+	MessageTypes.registerType({
+		id: 'video-direct-ended',
+		system: true,
+		message: 'video_direct_ended',
+	});
+	MessageTypes.registerType({
+		id: 'video-conference-started',
+		system: true,
+		message: 'video_conference_started',
+		data(message) {
+			return {
+				conference: (isVideoConferenceMessage(message) && message.videoConference.name) || '',
```

Comment:
should we provide something in case the title is empty? right now it would render:
`diego.sampaio started  video conference`

not sure this will look good..

Reply:
I've discussed with the design team and we'll keep it empty as it'll only be empty in case of error.